package.json

```diff
@@ -86,12 +86,18 @@
     "vitest": "3.1.4",
     "vue": "3.5.16",
     "vue-eslint-parser": "^10.1.1",
-    "vue-popperjs": "github:bethinkpl/vue-popper#03c7912c4729386743b0cca8f39b35448cc3db7f",
-    "vue-tsc": "2.2.10"
+    "vue-popperjs": "github:bethinkpl/vue-popper#df07fdb562d604fad966f0e8d6db1fc48febad6e",
```

Comment:
Migrated vue-popper to ESM as well - we should update the commit hash after we merge https://github.com/bethinkpl/vue-popper/pull/2